controllers/hosting/hostingdeployment_controller_test.go

```diff
@@ -217,15 +217,11 @@ var _ = Describe("Reconciling a HostingDeployment that exists", func() {
 	})
 
 	Context("Endpoint does not exist", func() {
-
-		BeforeEach(func() {
-			mockSageMakerClientBuilder.
-				AddDescribeEndpointErrorResponse(clientwrapper.DescribeEndpoint404Code, clientwrapper.DescribeEndpoint404MessagePrefix, 400, "request id")
-		})
-
 		Context("HasDeletionTimestamp", func() {
-
 			BeforeEach(func() {
+				mockSageMakerClientBuilder.
```

Comment:
Just cleanup ?

Reply:
Yes had to create a new context to use the deployment with the `endpointName`, so most of the removes here are just me moving code around to fit.